web/revping.go

```
@@ -0,0 +1,48 @@
+package web
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/go-ping/ping"
```

golangci-lint (1.62.2)

SA1019: "github.com/go-ping/ping" is deprecated: This package is no longer maintained.

(staticcheck)